src/tests/test_jsproxy.py

```diff
@@ -2249,8 +2250,8 @@ def test_python_reserved_keywords(selenium):
     assert o.async_ == 1
     assert o.async__ == 2
     assert o.async___ == 3
-    assert getattr(o, "async_") == 1
-    assert getattr(o, "async__") == 2
+    assert o.async_ == 1
+    assert o.async__ == 2
```

Comment:
These tests need to be kept as-is.

Reply:
Well the assertions here are a bit paranoid they could perhaps be removed. But they should either be removed or put back.